### PR TITLE
Add login page

### PR DESCRIPTION
**Pull request by serqiiiio**
-add Google logging Oauth 2 
- Create the Loin Page 
- After login, redirect to home page

### CODE REVIEW

File: RESEARCH/teseractResearch/tesseract.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+const worker = createWorker();
+(async () => {
+    await worker.load();
+    await worker.loadLanguage("eng");
+    await worker.initialize("eng");
+    const {
+        data: { text }
+    } = await worker.recognize(
+        //"https://telegram.org/file/811140100/2/maZcBXgwrmE.306486/5bd7c8f4708afe28f8"
```
Comment (virichino) [inline RESEARCH/teseractResearch/tesseract.js:12]: Delete comments

**Review by stellith — APPROVED** [review]

File: ReactUI/src/ui/pages/login/LoginPage.jsx [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+	CssBaseline,
+	Typography,
+	TextField,
+	Button,
+	Divider,
+	Grid,
+} from "@material-ui/core";
+
+import GoogleLogin from "react-google-login";
+
```
Comment (virichino) [inline ReactUI/src/ui/pages/login/LoginPage.jsx:15]: Delete unnecessary break line

File: ReactUI/src/ui/pages/login/LoginPage.jsx [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+	Paper,
+	Avatar,
+	CssBaseline,
+	Typography,
+	TextField,
+	Button,
+	Divider,
+	Grid,
+} from "@material-ui/core";
+
```
Comment (virichino) [inline ReactUI/src/ui/pages/login/LoginPage.jsx:13]: Delete unnecessary break line

File: ReactUI/src/ui/pages/login/LoginPage.jsx [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+		font: {
+			color: "#11a0bd",
+		},
+	}));
+	const classes = classStyles();
+
+	return (
+		<div className="Login">
+			<video src="background.mp4" autoPlay loop muted></video>
+
```
Comment (virichino) [inline ReactUI/src/ui/pages/login/LoginPage.jsx:56]: Same here, delete unnecessary break line

File: ReactUI/src/ui/pages/MachineLearning/MachineLearningPage.jsx [hunk 5]
```diff
@@ -1,8 +1,10 @@
 import React from "react";
+import { Redirect } from "react-router";
+
```
Comment (virichino) [inline ReactUI/src/ui/pages/MachineLearning/MachineLearningPage.jsx:3]: Delete unnecessary break line

**Review by virichino — COMMENTED** [review]
Check that the tab uses only four spaces

**Pull request merged**